### ISSUE TITLE
Deprecate SDK-docs repo

### ISSUE THREAD

**Issue by edd** [vegaprotocol/protos#237]
https://github.com/vegaprotocol/sdk-docs is a repo we have that appears to me to no longer be required. This ticket sums up the few things that are still required, in other repos, before we can get rid of this repo.

# History
`sdk-docs` was created to move generated documentation out of the core. The documentation was generated in order to populate the API section of https://docs.fairground.vega.xyz/docs/api-reference/. This is now being replaced with a new set of documentation (see https://github.com/vegaprotocol/documentation/).

# What is still needed
* A JSON representation of the protobufs (https://github.com/vegaprotocol/sdk-docs/blob/develop/generated/doc.json). This is used by https://github.com/protobuffet/docusaurus-protobuffet to generate the GRPC documentation
* OpenAPI/Swagger doc - This is used by https://github.com/cloud-annotations/docusaurus-openapi To generate the REST documentation section

# What is no longer needed
* `graphql.schema` - for `console`, it can be fetched from https://github.com/vegaprotocol/data-node/blob/develop/gateway/graphql/schema.graphql
* Generated GraphQL HTML documentation - this is replaced by https://github.com/edno/graphql-markdown
* Generated GRPC HTML documentation - see above

# What we need to do to deprecate the repo
Each time a proto release is tagged, we need to generate the following and have them available somewhere that the docusaurus docs repo can fetch them:
- [ ] Generate a JSON representation of the protos
- [ ] Generate an OpenAPI v3 yaml
- [ ] https://github.com/vegaprotocol/documentation/issues/44
- [ ] Hit the `Archive repository button`

**Comment by tguinot** [comment]
JSONs were already generated by the swagger (open api v2)
At the moment there is no hackish way of generating the v3 but we should be able to get docusaurus to work with v2
